tests/test_ops/test_ops.py

```diff
@@ -379,6 +379,85 @@ def wrapped_function(boxes, scores):
             save_dir=save_dir)
 
 
+@pytest.mark.parametrize('backend', [TEST_TENSORRT])
+@pytest.mark.parametrize('num_classes,pre_topk,after_topk,iou_threshold,'
+                         'score_threshold,background_label_id',
+                         [(5, 6, 3, 0.7, 0.1, -1)])
+def test_batched_rotated_nms(backend,
+                             num_classes,
+                             pre_topk,
+                             after_topk,
+                             iou_threshold,
+                             score_threshold,
+                             background_label_id,
+                             input_list=None,
+                             save_dir=None):
+    backend.check_env()
+    import importlib
+    if importlib.util.find_spec('mmrotate') is None:
```

Comment:
could use `pytest.importorskip('mmrotate', reason='mmrorate is not installed.')`